### ISSUE TITLE
FeatureHasher not working

### ISSUE THREAD

**Issue by mkrasmus** [HEMANGANI/Enrollment-Forecast#1]
Hi thank you for this incredible example that I'm finding useful.

Wondering if you might know what is going on with the following code chunk:
# Program Grouping: 147 to 8 features
from sklearn.feature_extraction import FeatureHasher
fh = FeatureHasher(n_features=8, input_type='string')
hashed_features = fh.fit_transform(train_data['Program Grouping'])
hashed_features = hashed_features.toarray()
hashed_features.shape
train_data=pd.concat([pd.DataFrame(train_data),pd.DataFrame(hashed_features)],axis=1)
train_data.rename(columns={0:'pg_0', 1:'pg_1', 2:'pg_2', 3:'pg_3', 4:'pg_4', 5:'pg_5', 6:'pg_6', 7:'pg_7'}, inplace=True)
train_data = train_data.drop('Program Grouping', axis=1)
train_data.head()

resulting in this error:
Samples can not be a single string. The input must be an iterable over iterables of strings.

The column train_data['Program Grouping'] is a string vector but not sure if this itself should be hashed or the intention of this part of your script. Any advice would be highly appreciated.
